geom/src/polygon.rs

```diff
@@ -329,7 +327,7 @@ impl Polygon {
     /// Usually m^2, unless the polygon is in screen-space
     pub fn area(&self) -> f64 {
         // Polygon orientation messes this up sometimes
-        to_geo(&self.points()).area().abs()
+        to_geo(&self.points()).signed_area().abs()
```

Comment:
Since https://github.com/georust/geo/pull/463 we now have unsigned area.

I'd *guess* that's what we want? (And then we could get rid of the incriminating comment. 😄

Reply:
Sounds good!